### PR TITLE
Add "Storing Data in the SPTPersistentCache"

### PR DESCRIPTION
**Pull request by 8W9aG**
- Adds another code example section to the usage
  examples

### CODE REVIEW

**Comment by 8W9aG** [comment]
@spotify/objc-dev

File: README.md [hunk 1]
```diff
@@ -72,6 +72,19 @@ SPTPersistentCache *cache = [[SPTPersistentCache alloc] initWithOptions:options]
 ```
 Note that in  the above example, the `currentTimeCallback` is `nil`. When this is nil it will default to using `NSDate` for its current time.
 
+### Storing Data in the SPTPersistentCache
+When storing data in the `SPTPersistentCache`, you must be aware of the file system semantics. The key will be used as the file name within the cache directory to save. The reason we did not implement a hash function under the hood is because we wanted to give the option of what hash function to use to the user, so it is recommended that when you insert data into the cache for a key, that you create the key using your own hashing function (at Spotify we use SHA1, although better hashing functions exist these days). If you want the file to exist outside of any TTL you set make sure you store it as a locked file.
```
Comment (rastersize) [inline README.md]: The quoted lined is difficult to read, so how about changing:

> If you want the file to exist outside of any TTL you set make sure you store it as a locked file.

to

> If you want the cache record, i.e. file, to exist without any TTL make sure you store it as a locked file.

Comment (8W9aG) [inline README.md]: :+1: Will make the change

**Comment by rastersize** [comment]
:+1:

**Pull request merged**